apps/dsp-app/cypress/support/helpers/constants.ts

```diff
@@ -0,0 +1,36 @@
+export enum ClassType {
+  Resource = 'Resource',
+  StillImageRepresentation = 'StillImageRepresentation',
+  MovingImageRepresentation = 'MovingImageRepresentation',
+  AudioRepresentation = 'AudioRepresentation',
+  TextRepresentation = 'TextRepresentation',
+  DocumentRepresentation = 'DocumentRepresentation',
+  ArchiveRepresentation = 'ArchiveRepresentation',
+}
+

```

Comment:
can't we import those enums / interfaces ? Don't they exist somewhere in the project ?

Reply:
I think those comes from the backend. Anyhow, its ok to have strings unrelated with API in e2e tests.